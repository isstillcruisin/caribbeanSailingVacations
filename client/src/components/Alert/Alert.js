import React from 'react'

function Alert(props) {
  if (props.alert) {
    return (<div className='alert'>{props.alert}</div>)
  } else {
    return ''
  }
}

export default Alert