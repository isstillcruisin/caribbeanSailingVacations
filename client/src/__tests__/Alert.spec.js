import React from "react";
import { create } from "react-test-renderer";
import Alert from '../components/Alert';

describe("Alert component", () => {
  test("Matches the snapshot", () => {
    const alert = create(<Alert alert='test' />);
    expect(alert.toJSON()).toMatchSnapshot();
  });
});