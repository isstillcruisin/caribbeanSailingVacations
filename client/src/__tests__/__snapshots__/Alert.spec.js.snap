// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert component Matches the snapshot 1`] = `
<div
  className="alert"
>
  test
</div>
`;
